cmd/cluster-kube-apiserver-operator/render/render.go

```diff
@@ -190,3 +199,21 @@ func (r *renderOpts) configFromDefaultsPlusOverride(tlsOverride string) ([]byte,
 
 	return yml, nil
 }
+
+func readFileTemplate(fname string, data interface{}) ([]byte, error) {
```

Comment:
didn't you make a library-go function for this?

Reply:
Don't think that makes sense. The whole render command plumbing belongs into a library-go package though.